CHANGELOG.md

```diff
@@ -1 +1,29 @@
-## TODO ADD CHANGELOGS MANUALLY FOR FIRST RELEASE
\ No newline at end of file
+# Release v0.1.0
+<img width="300" align="right" alt="OpenEBS Logo" src="https://raw.githubusercontent.com/cncf/artwork/master/projects/openebs/stacked/color/openebs-stacked-color.png" xmlns="http://www.w3.org/1999/html">
+
+---
+## Notes
+OpenEBS-CTL v0.1.0 is a first feature release.<br/> 
+Users are encouraged to install this tool, use it and help us know what can be better.<br/>
+Thank you to all that contributed with flushing out issues with OpenEBS-CTL!<br/>
+You can checkout the [documentation](https://github.com/openebs/openebsctl#readme) for more information.<br/>
+
+---
+## Notable Changes
+We have added supoort for `cStor` storage engine.<br/>
+* cStor `volume` and `pools` listing commands
+* cStor `volume` and `pools` describing commands
+* `PersistentVolumeClaim` describe command
+
+## What's Next
+* Restructuring of code, to add supoort for other Storage Engines
+* Adding support for Jiva
```

Comment:
Let us merge the jiva PR and include that into this release. 

Also, for the future list include these along with the refactoring change:
* Adding support for Local PV
* Adding support for Mayastor
* Adding support for managing Storage Devices
* Adding support for performing sanity checks and flagging discrepancies like listing stale volumes or over-utilized pools. 
* Adding support for getting overall status (like kubectl openebs clusterinfo)
* Ability to generate raise github issues with required  troubleshooting information.

Reply:
Done. Added these in the `What's Next`